build/Common.props

```
@@ -25,7 +25,7 @@
     -->
     <MicrosoftSourceLinkGitHubPkgVer>[1.0.0,2.0)</MicrosoftSourceLinkGitHubPkgVer>
     <!-- 0.5+ -->
-    <OpenFeatureVer>[1.2,)</OpenFeatureVer>
+    <OpenFeatureVer>1.4.0</OpenFeatureVer>
```

```suggestion
    <OpenFeatureVer>[1.4,)</OpenFeatureVer>
```

Should we keep the 1.4 version as minor and let it bump naturally? Also, we might need to bump again when 2.0 comes with the breaking changes.

---

This makes sense to me

---

IMO floating package versions haunt my nightmares. I've been burned more than once by a floating package version causing a lower version to get resolved from a CI build server cache.

That doesn't really apply in the era of cloud-hosted CI servers though, so will revert...